### PR TITLE
Load multiple fixtures at once

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets |
| License         | MIT

In current `nelmio/alice` version, if there is a reference to another entity which is not in the same fixture file, these files needs to be loaded together.

### CODE REVIEW

File: test/app/config/doctrine/Category.orm.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                strategy: AUTO
+    fields:
+        name:
+            type: string
+    manyToMany:
+        products:
+            targetEntity: Lakion\ApiTestCase\Test\Entity\Product
+            cascade: ["all"]
+            joinTable:
+                name: gus_category_products
```
Comment (lchrusciel) [inline test/app/config/doctrine/Category.orm.yml]: `gus` -> `app`

Comment (Zales0123) [reply test/app/config/doctrine/Category.orm.yml]: xD

**Comment by lchrusciel** [comment]
Thanks Mateusz! 🐃

**Comment by Zales0123** [comment]
Thank you Łukasz! Is there any chance for a patch release in the nearest future? 😄 I'm blocked by this in one of my Sylius PR's (see above).

**Comment by lchrusciel** [comment]
Done ;)

**Comment by pamil** [comment]
Psst, it looks like a candidate for minor release more than for patch release :)

File: src/ApiTestCase.php [hunk 2]
```diff
@@ -285,6 +285,25 @@ protected function loadFixturesFromFile($source)
...
     }
 
+    /**
+     * @param array $sources
+     *
+     * @return array
+     */
+    protected function loadFixturesFromFiles(array $sources)
+    {
+        $realPaths = array();
```
Comment (pamil) [inline src/ApiTestCase.php]: `[]` - PHP 5.4+ is the future! 🎉

**Pull request merged**